attachment_queue/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2015 Florian DA COSTA @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Attachment Queue",
+    "version": "12.0.1.0.0",
+    "author": "Akretion,Odoo Community Association (OCA)",
+    "summary": "Base module that add the concept of queue for processing file",
```

Comment:
s/that adds/that adds/
otherwise that's an English mistake. But I would replace it by "adding"

Reply:
I will fix here